ex1/dates_times.py

```diff
@@ -33,12 +33,49 @@
 # Define a function named last_of_month that takes an argument dt of type date
 # and returns a date object representing the last day of the month dt was in.
 
+import datetime
+
+def last_of_month(dt):
+    """Return a date object with the last day of the month of the provided date
+    """
+    dt = dt.replace(day=1) + datetime.timedelta(days=31)
+    dt= dt.replace(day=1)
+    return dt - datetime.timedelta(days=1)
```

Comment:
Not necessary to implement this. `calendar.monthrange` can do that for you. See https://github.com/TUW-GEO-python-intro/Exercise-1/pull/20/files#diff-b5dc1ec9f5b885f64b8dbe958d8d4a56R54